### DISCUSSION TITLE
R18版本疑问

### DISCUSSION THREAD

**Discussion by MFuching** [gedoor/gedoor.github.io#11] [Q&A] [answered]
R18版本有必要安装吗？我是指，所谓的普通版本会屏蔽部分网站是导入书源的时候被屏蔽掉吗？因为我不想被屏蔽这些网站，但是我看到R18版本更新过于频繁了，每天一到两更，而且好像不能在APP里自动更新。

**Comment by heziyouyi** [comment, marked as answer]
统一回复：什么是R18版，和酷安上架的版本，Github release的版本以及Play版(包括pro版，2.0和3.0)有何区别。

由于某些因素，阅读在上架国内应用市场时需要屏蔽常见的黄色书籍网站，因此加入了屏蔽内容，酷安上架的3.0以及Github release下的3.0和2.0均屏蔽了这些内容，而上架Google Play的版本（pro版，2.0，3.0）并无此限制。

R18共有两个版本，共存版为开发者在Github Action内跑的版本，去除了这个限制，并且由于包名不同可以和原版共存，而原包名是 @wushidi 基于Github release反编译修改的版本，一样去除了该限制，但由于签名和开发者的不一致，普通用户安装时需要卸载原版再安装（注意备份）。

给想知道如何修改的用户：
反编译安装包，将 assets/18PlusList.txt 中的内容清空（内容为base64编码）并回打包即可。

**Reply by MFuching** [reply]
事实上这段说明我已在频道内看过，但是不甚详细所以我又提出问题。我想搞清楚的两点：一是屏蔽R18网站发生在何阶段（是导入书源的时候就被屏蔽还是能导入但是不能访问），二是如此高的更新频率有些望而生畏，并且我昨晚在七点多GitHub更新后尝试在软件内检查更新但是检测不到更新，所以疑虑软件不能自动更新那就有些麻烦了毕竟更新实在频繁。
        我从昨晚开始已经在使用原包名版，上述两个问题，如果能回答非常感谢，不行的话，我再观察观察，或许昨晚只是因为更新不久暂未检测到。
